packages/webapp/src/App.jsx

```diff
@@ -16,20 +16,27 @@
 import { Suspense } from 'react';
 import Navigation from './containers/Navigation';
 import history from './history';
-import Routes from './Routes.jsx';
 import clsx from 'clsx';
 import { SnackbarProvider } from 'notistack';
 import { NotistackSnackbar } from './containers/Snackbar/NotistackSnackbar';
 import { OfflineDetector } from './containers/hooks/useOfflineDetector/OfflineDetector';
 import SlideMenu from './containers/Navigation/SlideMenu';
 import styles from './styles.module.scss';
+import { useMediaQuery } from '@mui/material';
+import { useTheme } from '@mui/styles';
+import Routes from './routes';
 
 function App() {
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up('sm'));
+
   return (
     <div className={clsx(styles.container)}>
-      <Suspense fallback={null}>
-        <SlideMenu history={history} classes={{ container: styles.slideMenu }} />
-      </Suspense>
+      {isDesktop && (
+        <Suspense fallback={null}>
+          <SlideMenu history={history} classes={{ container: styles.slideMenu }} />
```

Comment:
It would be cool if we could move this into the Navigation component. Then the extra suspense wouldn't be needed and it would all be in one place.

Reply:
You're not wrong since this is actual navigation, but I'm hesitant to do this much refactoring at this point to make re-reviews a bit easier, so I'll leave it as is for now!

Reply:
Sounds good 👍